catalyst_voices/lib/widgets/snackbar/voices_snackbar.dart

```diff
@@ -0,0 +1,146 @@
+import 'package:catalyst_voices/widgets/snackbar/voices_snackbar_type.dart';
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_brands/catalyst_voices_brands.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:flutter/material.dart';
+
+class VoicesSnackBar extends StatelessWidget {
+  final VoicesSnackBarType snackBarType;
```

Comment:
What about calling it just 'type'? It's used inside the VoicesSnackBar so it's clear what it is used for